File: 백준/Bronze/10989. 수 정렬하기 3/수 정렬하기 3.py
#BOJ - 10989 수 정렬하기 3
import sys

N = int(sys.stdin.readline())
li = [0] * 100001 # 0 ~ 10000

for _ in range(N):
    num = int(sys.stdin.readline())
    li[num] = li[num] + 1 #배열에는 각 숫자의 개수가 들어가게 된다.

for i in range(10001):
    if li[i] != 0 :
        for j in range(li[i]):
            print(i)#숫자의 수 만큼 출력

File: 백준/Bronze/10989. 수 정렬하기 3/수 정렬하기 3.java
import java.io.*;
import java.util.Arrays;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw = new BufferedWriter((new OutputStreamWriter(System.out)));
        int N = Integer.parseInt(br.readLine());

        int[] arr = new int[N];

        for(int i = 0 ; i < N ; i++){
            int num = Integer.parseInt(br.readLine());
            arr[i] = num;
        }

        Arrays.sort(arr);

        for(int n : arr){
            bw.write(n+"\n");
        }

        br.close();
        bw.flush();
        bw.close();

    }
}
